CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Windows stability fixes.
 * Mock module resolution fixes.
 * Remove test files from code coverage.
+* Add support for property testing via testcheck-js.
```

Comment:
can you move this to the `## master` section? Otherwise I usually compile the list before release. When many people work on Jest and they all modify the changelog, it often results in merge conflicts.